src/systems/ec/implementations/react/packages/inpage-navigation/src/InpageNavigation.jsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Link from '@ecl/ec-react-component-link';
+import Icon from '@ecl/ec-react-component-icon';
+
+const InpageNavigation = ({ title, links, className, ...props }) => {
+  const classNames = classnames(className, 'ecl-inpage-navigation');
+  return (
+    <nav {...props} className={classNames} data-ecl-inpage-navigation-sticky>
```

Comment:
why is it `data-ecl-inpage-navigation-sticky` and not just `data-ecl-inpage-navigation`? :)